doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2453 @@
+
+It's easy to save and load and array with **np.save()**. Just make sure to specify the array you want to save and a file name.  For example, if you create this array:
+
+::
+
+  a = np.array([1, 2, 3, 4, 5, 6])
+
+You can save it as "filename.npy" with
+
+::
+
+  np.save('filename',a)
+
+You can use **np.load()** to reconstruct your array.
+
+::
+
+  b = np.load('filename.npy')
+
+If you want to check your array, you can run:
+
+**Input:**
+
+::
+
+  print(b)
+
+**Output:**
+
+::
+
+  [1 2 3 4 5 6]
+
+
+You can save a NumPy array as a plain text file like a **.csv** or **.txt** file with **np.savetxt**.
+
+For example, if you create this array:
+
+::
+
+  csv_arr = np.array([1, 2, 3, 4, 5, 6, 7, 8])
+
+You can easily save it as a .csv file with the name "new_file.csv" like this:
+
+::
+
+  np.savetxt('new_file.csv', csv_arr)
+
+You can quickly and easily load your saved text file using **loadtxt()**:
+
+**Input:**
+
+::
+
+  np.loadtxt('new_file.csv')
+
+**Output:**
+
+::
+
+  array([1., 2., 3., 4., 5., 6., 7., 8.])
+
+
+The **savetxt()** and **loadtxt()** functions accept additional optional parameters such as header, footer, and delimiter. While text files can be easier for sharing, .npy and .npz files are faster to retrieve. If you need more sophisticated handling of your text file (for example, if you need to work with lines that contain missing values), you will want to use the  `genfromtxt function <https://docs.scipy.org/doc/numpy/reference/generated/numpy.genfromtxt.html#numpy.genfromtxt>`_.
+
+With savetxt, you can specify headers, footers, comments, and more. `Read more about savetxt here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.savetxt.html>`_.
+
+You can read more about `save <https://docs.scipy.org/doc/numpy/reference/generated/numpy.save.html>`_ here, `savez <https://docs.scipy.org/doc/numpy/reference/generated/numpy.savez.html>`_ here, and `load <https://docs.scipy.org/doc/numpy/reference/generated/numpy.load.html>`_ here. 
+You can read more about `savetxt <https://docs.scipy.org/doc/numpy/reference/generated/numpy.savetxt.html>`_ here, and `loadtxt <https://docs.scipy.org/doc/numpy/reference/generated/numpy.loadtxt.html>`_ here.
+
+Learn more about `input and output routines here <https://docs.scipy.org/doc/numpy/reference/routines.io.html>`_.
+
+**Be aware that loading files that contain object arrays with np.load() uses the pickle module which is not secure against erroneous or maliciously constructed data. Consider passing allow_pickle=False to load data that is known not to contain object arrays for the safer handling of untrusted sources.**
```

Comment:
I'd remove this, that doesn't really belong in an intro tutorial. It's a case that doesn't really happen in practice.

Reply:
Done!